setup.cfg

```diff
@@ -3,6 +3,7 @@ name = pylxd
 version = 2.3.2a
 description = python library for LXD
 long_description = file: README.rst
+long_description_content_type = text/markdown
```

Comment:
Since the `README.rst` is of type reStructuredText, shouldn't we proactively specify `long_description_content_type = text/x-rst` to remove the warning?

Reply:
Hmm, oops, thanks for spotting!

Reply:
@roosterfish this error I introduced was indeed breaking the listing as we can see on the TestPyPi page: https://test.pypi.org/project/pylxd/
Thanks again!